### DISCUSSION TITLE
Edit profile through the API?

### DISCUSSION THREAD

**Discussion by paramt** [community/community#24826] [Apps, API and Webhooks]
<p>I was trying to change my GitHub profile bio through the GitHub API but couldn’t find anything regarding profile editing on the docs… Will this be possible anytime in the future?</p>
<p>Cheers</p>

**Comment by parithiban** [comment, marked as answer]
<p><a class="mention" href="/paramt">@paramt</a> </p>
<p>If you are using GITHUB V3 you can update it via.</p>
<p><a href="https://developer.github.com/v3/users/#update-the-authenticated-user" rel="nofollow noopener">https://developer.github.com/v3/users/#update-the-authenticated-user</a></p>
<p>You need to do a PATCH request</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/c/c7c162dcb6f2839dae2c84cea1236efacb0bc6b5.png" data-download-href="/uploads/short-url/sv7lmDNy5NQIEVS9zUbyZh5nfEh.png?dl=1" title="Screen Shot 2019-10-29 at 11.02.41 AM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/7281166/181087012-4910c18b-9347-49c7-be83-d73a4f28ee48.png" alt="Screen Shot 2019-10-29 at 11.02.41 AM.png" data-base62-sha1="sv7lmDNy5NQIEVS9zUbyZh5nfEh" width="690" height="277" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/c/c7c162dcb6f2839dae2c84cea1236efacb0bc6b5_2_690x277.png, https://aws1.discourse-cdn.com/github/optimized/2X/c/c7c162dcb6f2839dae2c84cea1236efacb0bc6b5_2_1035x415.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/c/c7c162dcb6f2839dae2c84cea1236efacb0bc6b5_2_1380x554.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/c/c7c162dcb6f2839dae2c84cea1236efacb0bc6b5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2019-10-29 at 11.02.41 AM.png</span><span class="informations">2266×912 115 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by paramt** [comment]
<p>Thanks, that’s exactly what I was looking for!</p>
